frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -172,7 +180,15 @@ function BookInfo:show(doc_settings_or_file, book_props)
     table.insert(kv_pairs, { _("Rating:"), ("★"):rep(rating) .. ("☆"):rep(self.rating_max - rating),
         hold_callback = summary_hold_callback })
     table.insert(kv_pairs, { _("Review:"), summary.note or _("N/A"),
-        hold_callback = summary_hold_callback })
+        hold_callback = summary_hold_callback, separator = true })
+
+    -- Notes file
+    local notes_file = has_sidecar and doc_settings_or_file:readSetting("notes_file")
+    local notes_file_callback = function()
+        self:showNotesFileDialog(notes_file, doc_settings_or_file, book_props)
+    end
+    table.insert(kv_pairs, { _("Notes file:"), notes_file and notes_file:gsub(".*/", "") or _("Tap to set"),
+        callback = notes_file_callback })
```

Comment:
(Each time you add an item to Book information, I need to increase my `Info list items per page` to have it neat on a single page. Text will become smaller and me blind :/ You're so evil :))

To all: no problem or confusion with naming that feature "notes file" ? We already have `Add note / Edit note` for associating our own text to a highlight, so this might be confusing.
No other idea about an alternative name for this feature.

Reply:
Notebook file might be slightly more distinct?